packages/experimental/TabList/src/Tab/Tab.styling.ts

```diff
@@ -100,7 +100,7 @@ export const useTabSlotProps = (props: TabProps, tokens: TabTokens, theme: Theme
     // if we're the selected tab and we've generated styles for the animated indicator, render the static tab indicator as transparent.
     // The animated indicator will receive styling instead via useTabAnimation hook.
     () => {
-      const hideStaticIndicator = props.tabKey === selectedKey && !!context.animatedIndicatorStyles;
+      const hideStaticIndicator = props.tabKey === selectedKey && indicatorHasRendered;
```

Comment:
`indicatorHasRendered` is confusing to me here, it's not clear it's talking about the animatedIndicator. Can we rename this to something more obvious like "canShowAnimatedIndicator"?